core/src/main/java/org/fao/geonet/kernel/datamanager/IMetadataOperations.java

```diff
@@ -0,0 +1,44 @@
+    package org.fao.geonet.kernel.datamanager;
```

Comment:
Just a question, obviously not critical though: why package is indented ?

Reply:
Probably eclipse misunderstood something...